### PR TITLE
Tap1.5 RA update 

### PR DESCRIPTION
**Pull request by sendjainabhi**
TAP 1.5 Reference architecture update 
The following items have been added 
- TAP Networking
-Update TAP RA for v 1.5

### CODE REVIEW

File: src/reference-designs/tap-architecture-planning.md [hunk 1]
```diff
@@ -193,13 +200,14 @@ The View Cluster's requirements are driven primarily by the respective applicati
 
 ### Kubernetes Requirements - View Cluster
 
-* Supported Kubernetes versions are 1.23, 1.24, 1.25.
-* LoadBalancer for ingress controller (requires 3 external IP addresses).
+* Supported Kubernetes versions are 1.24,1.25,1.26.
+* LoadBalancer for ingress controller (requires 1 external IP addresses).
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-planning.md:204]: ... 1 ... IP address).

File: src/reference-designs/tap-architecture-dev-components.md [hunk 2]
```diff
@@ -35,13 +35,13 @@ The Application Accelerator user interface (UI) enables you to discover availabl
...
 
-Refer to the [Tanzu Application Platform Accelerator](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/application-accelerator-about-application-accelerator.html) for more information.
+Refer to the [Tanzu Application Platform Accelerator](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/application-accelerator-about-application-accelerator.html) for more information.
 
 ## API Portal
 
 API portal enables API consumers to find APIs they can use in their own applications. API portal assembles its dashboard and detailed API documentation views by ingesting OpenAPI documentation from the source URLs.
 
-Refer to the [Tanzu Application Platform API portal](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/api-portal-install-api-portal.html) for more information.
+Refer to the [Tanzu Application Platform API portal](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/api-portal-install-api-portal.html) for more information.
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-dev-components.md:44]: See the ...

File: src/reference-designs/tap-architecture-dev-components.md [hunk 3]
```diff
@@ -111,11 +111,11 @@ spec:
 
 The settings in `ClientRegistration` contain the redirectUrl pointing to a page in the end-user application to be redirected to after successful authentication. The settings here also reference the AuthServer by its pod’s labels on behalf of the end-user application.
 
-Refer to the [Tanzu Application Platform AppSSO](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/app-sso-about.html) for more information.
+Refer to the [Tanzu Application Platform AppSSO](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/app-sso-about.html) for more information.
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-dev-components.md:114]: See the ...

File: src/reference-designs/tap-architecture-dev-components.md [hunk 4]
```diff
@@ -127,11 +127,11 @@ API Auto Registration components are installed by the `run` and `full` profiles.
...
+- The API Auto Registration package configuration in `run` clusters [must be updated](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/api-auto-registration-usage.html#update-values) to include the Tanzu Application Platform GUI URL allowing it to register workload APIs into the GUI in the `view` cluster.
 - Workload configuration files must set the `register-api` property to true to enable this feature and include the `api_descriptor` parameter. For example: `apis.apps.tanzu.vmware.com/register-api: "true"`. 
 - To use the OpenAPI “TRY IT OUT” feature in Tanzu Application GUI, the Workload must configure Cross-origin Resource Sharing (CORS) to allow requests originating from the GUI.
 
-To learn more about API Auto Registration, refer to [API Auto Registration](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/api-auto-registration-about.html).
+To learn more about API Auto Registration, refer to [API Auto Registration](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/api-auto-registration-about.html).
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-dev-components.md:134]: ... Registration, see [API Auto ...

File: src/reference-designs/tap-architecture-dev-components.md [hunk 5]
```diff
@@ -215,6 +215,6 @@ grype:
 
 ## CI/CD Pipelines
 
-Tanzu Application Platform supports Tekton pipelines using `tekton-pipelines package`. It allows developers to build, test, and deploy across cloud providers and on-premises systems. Refer [Tekton documentation](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/tekton-tekton-about.html) for more information.
+Tanzu Application Platform supports Tekton pipelines using `tekton-pipelines package`. It allows developers to build, test, and deploy across cloud providers and on-premises systems. Refer [Tekton documentation](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/tekton-tekton-about.html) for more information.
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-dev-components.md:218]: ... See [Tekton documentation ...

File: src/reference-designs/tap-architecture-dev-components.md [hunk 6]
```diff
@@ -215,6 +215,6 @@ grype:
...
 
-To learn more about all Tanzu Application Platform components refer to [Component documentation](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/components.html).
+To learn more about all Tanzu Application Platform components refer to [Component documentation](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/components.html).
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-dev-components.md:220]: To learn more about the Tanzu Application Platform components, see [Component ..

File: src/reference-designs/tap-architecture-planning.md [hunk 7]
```diff
@@ -505,12 +525,12 @@ Two roles are for service accounts associated with the Tanzu Supply Chain:
 * `workload`
 * `deliverable`
 
-Refer to [Tanzu Application Platform authorization](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/authn-authz-overview.html) for more information.
+Refer to [Tanzu Application Platform authorization](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/authn-authz-overview.html) for more information.
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-planning.md:528]: See [Tanzu Application ....

File: src/reference-designs/tap-architecture-planning.md [hunk 8]
```diff
@@ -505,12 +525,12 @@ Two roles are for service accounts associated with the Tanzu Supply Chain:
...
 
 ## Developer tools (Inner-Loop)
 
 Tanzu Application Platform allows developers to quickly build and test applications and provide many in-built developer friendly platform capabilities. To learn more about these capabilities, refer to [Tanzu Application Platform Developer Components](tap-architecture-dev-components.md).
 
 ## Deployment Instructions
 
-For instructions on how to deploy this reference design, refer to [Deploy multi-cluster Tanzu Application Platform profiles](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/multicluster-installing-multicluster.html).
+For instructions on how to deploy this reference design, refer to [Deploy multi-cluster Tanzu Application Platform profiles](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/multicluster-installing-multicluster.html).
```
Comment (srinivasme21) [inline src/reference-designs/tap-architecture-planning.md:536]: ... reference design, see [Deploy ...

File: src/reference-designs/tap-networking.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+# Tanzu Application Platform Networking
+
+Tanzu Application Platform Networking diagram describes the Tanzu Application Platform most common cross clusters and external services scenarios networking in multi cluster kubernetes environment. It describes high level egress and ingress networking flows between the Tanzu Application Platform components and with the external services.
```
Comment (srinivasme21) [inline src/reference-designs/tap-networking.md:3]: The following Tanzu Application Platform networking diagram shows the most common cross-cluster and external-services networking scenarios in a multi cluster Kubernetes environment. The diagram shows the high level egress and ingress networking flows between the Tanzu Application Platform components and the external services.

File: src/reference-designs/tap-networking.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
+# Tanzu Application Platform Networking
+
+Tanzu Application Platform Networking diagram describes the Tanzu Application Platform most common cross clusters and external services scenarios networking in multi cluster kubernetes environment. It describes high level egress and ingress networking flows between the Tanzu Application Platform components and with the external services. 
+
+![Tanzu Application Platform networking](img/tap-architecture-planning/tap-networking.jpg)
+
+Table describe the Tanzu Application Platform networking flow, shows in above diagram.
```
Comment (srinivasme21) [inline src/reference-designs/tap-networking.md:7]: The following table describes the networking flow in the Tanzu Application Platform networking diagram.

**Review by srinivasme21 — CHANGES_REQUESTED** [review]

File: src/reference-designs/tap-networking.md (hunk 10 above)
Comment (sendjainabhi) [reply src/reference-designs/tap-networking.md]: Changes done.

**Review by srinivasme21 — APPROVED** [review]
LGTM

**Pull request merged**